packages/frontend/components/global/CookiesBanner/CookiesBanner.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+import * as Styled from "./CookiesBanner.styles";
+
+export default function CookiesBanner({ onAccept, onClose }: Props) {
+  return (
+    <Styled.Banner className="l-container-max a-bg-custom20" role="alert">
```

Comment:
I'd remove the `l-container-max` class here - it's very likely the color will extend edge to edge on the page. I'll double check with Lael 👍